spec/steps/medicaid/insurance_current_type_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::InsuranceCurrentType do
+  describe "Validations" do
+    context "each insured household member has an insurance type" do
+      it "is valid" do
+        member = create(:member,
+                        is_insured: true,
```

Comment:
generally prefer booleans to NOT start with `is`

Reply:
✅